config/initializers/redis.rb

```diff
@@ -0,0 +1,4 @@
+require "redis"
+
+uri = URI.parse(ApplicationConfig["REDISCLOUD_URL"])
+RedisClient = Redis.new(:host => uri.host, :port => uri.port, :password => uri.password)
```

Comment:
Let's try to use the other type of keyword arguments:

```ruby
Redis.new(host: uri.host, ...)
```

Reply:
Can do! This was a copy and paste from docs.

If we want to enforce this we should probably add a Rubocop for it.

Reply:
Good call, I'll take a look